doc-content/jbpm-docs/src/main/asciidoc/DesignBuildCases/case-management-install-and-login-to-showcase-proc.adoc

```diff
@@ -5,7 +5,7 @@ The Showcase application is included with the {PRODUCT} {PRODUCT_VERSION} distri
 
 .Prerequisites
 
-* {CENTRAL} and {KIE_SERVER} are installed in an {EAP} instance.
+* {CENTRAL} and {KIE_SERVER} are installed in a {EAP} instance.
```

Comment:
I am not sure why you changed the content from **_an {EAP}_** to **_a {EAP}_**?

Can you please explain it to me?

Reply:
Because if you look at the variable {EAP} when it is expanded it is 'Red Hat JBoss EAP'. This trips a lot of us up.